docs/integrations/code-conventions.md

```diff
@@ -713,6 +713,34 @@ timeline = IndicatorsTimeline(
 )
 ```
 
+### PollingConfiguration
+The `PollingConfiguration` is an optional object (available from Server version 6.2.0 and up). When provided to a `CommandResults` it transforms its result to a ***polling result***.
+By returning a ***polling result***, the command (or script) indicates to XSOAR that it cannot yet return the full result (likely because it's waiting for a remote process to finish execution).
```

Comment:
I don't think there is a need to explain how it works or how the server handles it.
Instead explain who and when should use it and for which purposes.